variables.tf

```diff
@@ -2,8 +2,53 @@
 # Input Variables
 ########################################################################################################################
 
-#variable "my_variable" {
-#  type        = string
-#  description = "A description of my variable"
-#  default     = "default_value"
-#}
+variable "resource_group_id" {
+  type        = string
+  description = "The ID of the resource group to use for the creation of the Enterprise Application Service instance (https://test.cloud.ibm.com/account/resource-groups)."
```

Comment:
its pointing to a staging doc. I'm not sure you need to point to any doc here tbh. Its fairly obvious from the var name and description what this should be.

Reply:
yep, just something remaining from first ease team changes I missed to remove, thank you for pointing it out